### ISSUE TITLE
Rebrandly shortner missing

### ISSUE THREAD

**Issue by UVLabs** [Codeinwp/tweet-old-post#215]
For some reasons it didn't ship with the latest version, shortner file is also not in plugin

**Comment by UVLabs** [comment]
@contactashish13 can you please update your fork with https://github.com/Codeinwp/tweet-old-post/tree/8.1.1 and send this pull request again against that branch?

**Comment by contactashish13** [comment]
@UVLabs it might be easier for you to resolve it within the confines of git by reopening the PR and then merging it against 8.1.1

**Comment by UVLabs** [comment]
@contactashish13 for some reason it shows no changes available: 

https://github.com/contactashish13/tweet-old-post/tree/issue-166

![image](https://user-images.githubusercontent.com/10324144/45630055-da326880-ba65-11e8-8e73-9d30964a64e2.png)

Can you investigate here? I'm not sure why this happened but you might have to send a totally new pull request

**Comment by selul** [comment]
yes, that's right, i have disabled the test for now and merged this.
